testgen/generators.go

```diff
@@ -862,6 +863,76 @@ var EthGetTransactionReceipt = MethodTests{
 	},
 }
 
+var EthGetBlockReceipts = MethodTests{
+	"eth_getBlockReceipts",
+	[]Test{
+		{
+			"get-block-0",
+			"gets receipts for block 0",
+			func(ctx context.Context, t *T) error {
+				return t.rpc.CallContext(ctx, nil, "eth_getBlockReceipts", "0x0")
```

Comment:
Can we verify the result against the expected value? I do this on all the test to guard against the filling client misinterpreting the query and spitting out the wrong receipts. The way I usually do this is access the actual `Blockchain` object in `T` (`t.chain`) and pull the values from the structure to compare against the RPC result.

The tests you wrote for `eth_getTransactionReceipt` did this correctly.

Reply:
🙆‍♂️

Reply:
please take another look, thanks :)